packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -537,11 +547,36 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
     }
   };
 
+  /**
+   * Reveals suggestions any time the user clicks on the input element
+   * without shifting focus.
+   */
+  protected onClick = (ev: React.MouseEvent<HTMLInputElement>): void => {
+    const input = this.input.current ? this.input.current.value : '';
```

Comment:
Only thing I might do, check to see if this.props.inputprops has an onclick already, and then call that. Makes it less likely we're breaking someone that has an onclick.

Reply:
Ooh, that's a very good point. I'll definitely change that shortly.